docs/sphinx/reference-frontend-api.rst

```diff
@@ -242,6 +242,12 @@ Functions
 
 ---------------------------------------
 
+.. function:: void obs_frontend_close_main_window(void)
+
+   Closes the main window.
```

Comment:
Perhaps some clarification here is in order. Like, would it prompt the user if there are active outputs, or ignore outputs.

Reply:
I added a line about active outputs. I didn't want to make things too wordy. I could have stated that behaviour is the same as the Exit button, but I think it's clearer to just state the behaviour.

Reply:
Currently looks good to me. However, I would prefer:
> Closes the main window**, as if the user clicked the Exit button**. If there are active outputs, the user will be prompted to confirm before closing.

Which may or may not be necessary, and I'll leave that opinion to you and the others.

Reply:
I'll leave it unresolved for now, so others can weigh in.